waku/v2/node/waku_node.nim

```diff
@@ -911,16 +906,15 @@ proc runDiscv5Loop(node: WakuNode) {.async.} =
 
   while node.wakuDiscv5.listening:
     trace "Running discovery loop"
-    ## Query for a random target and collect all discovered nodes
-    ## TODO: we could filter nodes here
-    let discoveredPeers = await node.wakuDiscv5.findRandomPeers()
-    if discoveredPeers.isOk():
-      ## Let's attempt to connect to peers we
-      ## have not encountered before
+    let discoveredPeersRes = await node.wakuDiscv5.findRandomPeers()
 
-      trace "Discovered peers", count=discoveredPeers.get().len()
+    if discoveredPeersRes.isOk:
```

Comment:
```suggestion
    if discoveredPeersRes.isOk():
```
nit, but i believe this is the style guide we're adopting

Reply:
maybe we should handle an error if `findRandomPeers` fails. An error log atleast

Reply:
sure thanks!